specification/communication/data-plane/CallAutomation/preview/2024-01-22-preview/communicationservicescallautomation.json

```diff
@@ -2485,15 +2834,41 @@
         }
       }
     },
-    "StartDialogRequest": {
-      "required": [
-        "dialog"
-      ],
+    "StartMediaStreamingRequest": {
       "type": "object",
       "properties": {
-        "dialog": {
-          "$ref": "#/definitions/BaseDialog",
-          "description": "Defines the dialog."
+        "operationCallbackUri": {
+          "description": "Set a callback URI that overrides the default callback URI set by CreateCall/AnswerCall for this operation.\r\nThis setup is per-action. If this is not set, the default callback URI set by CreateCall/AnswerCall will be used.",
+          "type": "string"
+        },
+        "operationContext": {
+          "description": "The value to identify context of the operation.",
+          "type": "string"
+        }
+      }
+    },
+    "StopMediaStreamingRequest": {
+      "type": "object",
+      "properties": {
+        "operationCallbackUri": {
+          "description": "Set a callback URI that overrides the default callback URI set by CreateCall/AnswerCall for this operation.\r\nThis setup is per-action. If this is not set, the default callback URI set by CreateCall/AnswerCall will be used.",
+          "type": "string"
+        },
+        "operationContext": {
+          "description": "The value to identify context of the operation.",
+          "type": "string"
+        }
+      }
+    },
+    "StartDialogRequest": {
```

Comment:
any changes to this dialog request?

Reply:
All these are generated from master branch. I did not make any changes but there could be changes made on beta3 branch earlier